app/utils/client_upgrade.js

```diff
@@ -22,7 +22,9 @@ export function checkUpgradeType(currentVersion, minVersion, latestVersion, logE
             }
         }
     } catch (error) {
-        logError(error.message);
+        if (logError) {
+            logError(error.message); //disabled currently for Realm migration from client_upgrade screen
```

Comment:
Why is this being kept?

Reply:
I removed the use of logError for the client upgrade screen but in testing found that in channel base there is a listener component that uses this utils function and also passes a logError.

Reply:
then maybe fix the test?